src/components/AuthenticatedLink.svelte

```diff
@@ -2,9 +2,11 @@
   import tippy from "./tippy";
 
   export let href;
+  export let label = null; // a prop for glean click events
+  export let type = null; // a prop for glean click events
 </script>
 
-<a {href}><slot /></a>
+<a {href} data-glean-label={label} data-glean-type={type}><slot /></a>
```

Comment:
I am not 100% sure how Svelte works with props passing. But, if there is a way to only set those properties if we are guaranteed to have a value for them (so they aren't incorrectly set as undefined or something) then that would be good. Unless Glean.js is checking to make sure those values aren't undefined? I forget how the Glean.js implementation looks

Reply:
Yep, sounds good 👍